mono/metadata/debug-mono-ppdb.c

```diff
@@ -184,9 +184,9 @@ mono_ppdb_load_file (MonoImage *image, const guint8 *raw_contents, int size)
 		stream.next_in = ppdb_data;
 		stream.avail_out = ppdb_size;
 		stream.next_out = data;
-		int res = inflateInit2 (&stream, -15);
+		int res = z_inflateInit2 (&stream, -15);
```

Comment:
Shouldn't this be conditional if we use bundled zlib or not?

Reply:
Probably.